lib/dl_auth_api_lib/dl_auth_api_lib_tests/conftest.py

```diff
@@ -98,7 +95,7 @@ def oauth_app(loop, aiohttp_client, oauth_app_settings):
     return loop.run_until_complete(aiohttp_client(app))
 
 
-@pytest.fixture(scope="function")
+@pytest_asyncio.fixture(scope="function")
```

Comment:
no idea why i need it locally for successful tests, investigating

Reply:
AFAIC `pytest_asyncio.fixture` is the correct and intended way of doing async fixtures, it was modified by `aiohttp.pytest_plugin`